db/seeds.rb

```diff
@@ -28,3 +28,12 @@
   harvester_user.save!(validate: false)
 end
 
+# default dataset
+default_dataset = Dataset.where(name: 'default').first
+if default_dataset.blank?
+  default_dataset = Dataset.new(name: 'default')
+  default_dataset.description = 'The default dataset'
+  default_dataset.creator_id = admin_user.id
```

Comment:
While `default_dataset.creator_id = admin_user.id` should work, it might be preferable style-wise to use `default_dataset.creator = admin_user`.